.github/workflows/main.yml

```diff
@@ -11,7 +11,9 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - name: Install Rust (rustup)
-      run: rustup update stable --no-self-update && rustup default stable
+      run: |
+        rustup update stable --no-self-update
+        rustup default stable
```

Comment:
Is `set -e` used? Aka does one failing command cause the execution to stop?

Reply:
It appears that `sh` is the default shell for Linux and macOS, which is `-e`.  Setting the shell to `bash` will set `-eo pipefail`.

I don't think PowerShell has any such concept, but I think it behaves like -e does (that is, it will exit with failure on the first cmdlet that throws an exception).

However, I think it would probably be best to set `bash` as the shell so that we don't run into future incompatibilities.